README.md

```diff
@@ -31,14 +31,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the DOM?
-
+- The DOM is the Document Object Model
 2. What is an event?
-
+- An event is an action such as clicking or pressing.
 3. What is an event listener?
-
+- It listens for things such as clicks, scrolling, key press, so it can do whatever the event listener calls for.
 4. Why would we convert a NodeList into an Array?
-
+- So you can iterate over it. 
```

Comment:
You can iterate over a node list with forEach. NodeList methods are limited, converting it to an array give you access to Array methods.